mitosheet/mitosheet/code_chunks/export_to_file_code_chunk.py

```diff
@@ -10,6 +10,30 @@
 from mitosheet.state import State
 from mitosheet.transpiler.transpile_utils import TAB, column_header_to_transpiled_code
 
+# This is a helper function that generates the code for formatting the excel sheet
+def get_format_code(state: State) -> list:
+    code = []
+    formats = state.df_formats
+    for sheet_name, format in zip(state.df_names, formats):
+        # If there is no formatting, we skip trying to access the colors
+        if format.get('headers') is None:
+            continue
+        
+        # Prepares the header font color and background color to be used in the API call
+        # So either they should be surrounded by quotes, or they should be None
+        header_font_color = None
+        header_background_color = None
+        if (format['headers'].get('color') is not None):
+            header_font_color = column_header_to_transpiled_code(format["headers"]["color"])
+        if (format['headers'].get('backgroundColor') is not None):
+            header_background_color = column_header_to_transpiled_code(format["headers"]["backgroundColor"])
+        
+        # If both are None, we skip this sheet
+        if header_font_color is None and header_background_color is None:
+            continue
+        code.append(f'{TAB}add_formatting_to_excel_sheet(writer, "{sheet_name}", header_background_color={header_background_color}, header_font_color={header_font_color})')
```

Comment:
Ok so one thing the `param_dict_to_code` could be useful for is not having to manually write all these out. 

```
>>> param_dict_to_code({'header_background_color': 'ffffff', 'header_font_color': 'xxxxx'}, as_single_line=True)
"header_background_color='ffffff', header_font_color='xxxxx'"
```

Let's move to this, I think. That way, we can just build a params dict and then pass it to this function without having to update variable names in the future.

Reply:
Just pushed to use that! How does that look?